api/data_explorer/controllers/export_url_controller.py

```diff
@@ -9,14 +9,18 @@
 import time
 import urllib
 
+from collections import OrderedDict
```

Comment:
I no longer get "Data too long for column" after taking out fastq, so let's do that. You're a Project Owner on verily-public-data, so I'll let you handle the switchover.

It might make sense to send/merge the 2 PRs making the switch before this PR.

Reply:
Removed fastq.